smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -126,6 +129,31 @@ function makeReaderNameForDisplaying(readerName) {
   return readerName;
 }
 
+/**
+ * @param {!GSC.PcscLiteServer.ReaderInfo} reader 
+ * @return {string}
+ */
+function getTooltipText(reader) {
+  let toolTipText;
+  switch (reader['status']) {
+    case GSC.PcscLiteServer.ReaderStatus.INIT:
+      toolTipText = 'initReaderTooltip';
```

Comment:
nit: We can just do `return 'initReaderTooltip'` here and in other cases - that's shorter and avoids the need in "toolTipText" and "break".

Reply:
ah that's true